src/bundle/Resources/config/graphql/Item.types.yaml

```
@@ -37,7 +37,7 @@ AbstractItem:
             _type:
                 description: "The item's Content type"
                 type: ContentType
-                resolve: '@=resolver("ContentTypeById", [value.getContentInfo().contentTypeId])'
+                resolve: '@=resolver("ContentTypeById", [value.contentInfo.contentTypeId])'
```

`resolver` is deprecated so I suggest using `query` for the new code. Not 100% sure about the syntax, please double-check (ref: https://github.com/overblog/GraphQLBundle/blob/0.14/UPGRADE.md#replace-resolver-expression-function).
```suggestion
                resolve: '@=query("ContentTypeById", value.contentInfo.contentTypeId)'
```